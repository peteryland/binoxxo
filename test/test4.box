1o5o2
5x
x4xx
7o
4x3x
2xo1o
2x
o3o
o6o
2x5x
