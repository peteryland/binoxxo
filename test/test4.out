_O_____O__
_____X____
X____XX___
_______O__
____X___X_
__XO_O____
__X_______
O___O_____
O______O__
__X_____X_

XOOXXOXOOX
OXXOXXOXOO
XOOXOXXOXO
XOXXOOXOOX
OXOOXXOXXO
OXXOXOOXOX
XOXXOOXOXO
OXOXOXOXOX
OXOOXOXOXX
XOXOOXOXXO

